src/drivers/stm32/tone_alarm/tone_alarm.cpp

```diff
@@ -537,6 +540,8 @@ ToneAlarm::rest_duration(unsigned rest_length, unsigned dots)
 void
 ToneAlarm::start_note(unsigned note)
 {
+	if (circuit_breaker_enabled("CBRK_BUZZER", CBRK_BUZZER_KEY)) { return; }
```

Comment:
Sorry for not noticing earlier: The param should be evaluated on start and a flag be kept to cache the state. This is running the param interface at a way too high rate. One of the downsides of simplified APIs 8).